### DISCUSSION TITLE
video downloadhelper

### DISCUSSION THREAD

**Discussion by mld68** [community/community#136119] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi i think last year I purchased this video downloadhelper. It was working great, I have 3 sites that I download movies from. Just about several weeks ago it just suddenly gets to a certain percentage then it just stops. it doesn't keep downloading anymore. it would be 60, or 20 or 45 etc. percent then just stops. I tried uninstalling it, resetting. but still nothing works. Help.

**Comment by LiteBrite82** [comment]
Hi @mld68 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
